add_gem_package.sh

```diff
@@ -138,22 +105,31 @@ if [[ -z $TEMPLATE_NAME ]] ; then
 	fi
 fi
 
-if [[ -z $REPO ]] ; then
-	if [[ $TEMPLATE_NAME == *_plugin ]] ; then
-		REPO="foreman-plugins-el8"
-	fi
-fi
-
 if [[ -z $PACKAGE_SUBDIR ]] ; then
-	if [[ $REPO == foreman-plugins-* ]] ; then
+	if [[ $TEMPLATE_NAME == *_plugin ]] ; then
 		PACKAGE_SUBDIR="plugins"
-	elif [[ $REPO == katello-* ]] ; then
-		PACKAGE_SUBDIR="katello"
 	else
 		PACKAGE_SUBDIR="foreman"
 	fi
 fi
 
+if [[ -z $MANIFEST_SECTION ]] ; then
+	case $PACKAGE_SUBDIR in
+		client)
+			MANIFEST_SECTION="foreman_client_packages"
+			;;
+		foreman)
+			MANIFEST_SECTION="foreman_core_packages"
```

Comment:
I copied this from the old method, but I don't think this is a very good group for it. On the other hand, I don't think there is a sane default which is why it's now an optional parameter.